spot-client/src/spot-remote/ui/views/remote-views/in-call.js

```diff
@@ -111,11 +114,16 @@ export class InCall extends React.Component {
         }
 
         const { meetingName } = parseMeetingUrl(inMeeting);
+        const phoneNumber = this.props.invitedPhoneNumber && parsePhoneNumberFromString(this.props.invitedPhoneNumber);
```

Comment:
Maybe this should be passed in already parsed, if easy? Or maybe a separate component that does the parsin?

Reply:
hmm this parsing is really UI formatting so I would rather pass the data as raw as possible and do the formatting in the UI component